src/pages/proposal.css

```diff
@@ -25,6 +32,12 @@
   }
 }
 
+@media (min-width: 1440px) {
+  .ProposalDetailPage__Description {
+    max-width: calc(100% - 284px - 284px);
```

Comment:
What is 284px? The sidebar? Maybe we can use a css variable?

Reply:
Yes, both sidebars have 284px. I can add it to the theme like `--proposal-sidebar-width`